csi/moac/test/csi_test.js

```diff
@@ -392,7 +424,7 @@ module.exports = function() {
               block: {},
             },
           ],
-          parameters: { repl: '3' },
+          parameters: { repl: '3', protocol: 'nexus_nbd' },
         });
         sinon.assert.calledWith(createVolumeStub, UUID, {
```

Comment:
can we add a check for return value of createVolume CSI method now when it actually matters? It should return `protocol = nbd` in the output params.

Reply:
I think this is done in the test case at line 217 "It should create a volume and return parameters in volume context"